federatedpersistence/src/main/java/org/openmbee/mms/federatedpersistence/permissions/FederatedPermissionsUpdateResponseBuilder.java

```diff
@@ -0,0 +1,99 @@
+package org.openmbee.mms.federatedpersistence.permissions;
```

Comment:
these were functions from PermissionsUpdateResponseBuilder that uses the DB classes